attestation/git/git_test.go

```diff
@@ -43,6 +43,25 @@ func TestNameTypeRunType(t *testing.T) {
 	require.Equal(t, RunType, attestor.RunType(), "Expected the attestor's run type")
 }
 
+func TestNormaliseRemotes(t *testing.T) {
+	remotes := []string{
+		"github.com/in-toto/go-witness",
+		"github.com/in-toto/go-witness.git",
+		"https://github.com/in-toto/go-witness",
+		"https://github.com/in-toto/go-witness.git",
```

Comment:
I think it may be worth testing with `git://` and `git+ssh` etc, even though some of those are deprecated. I think some SCPs (possibly sourceforge?) sets git:// to be default.

Reply:
a-ha - I will!